### ISSUE TITLE
Include both green and black olivada recipes

### ISSUE THREAD

**Issue by l-gar-alv** [cr-workshop-exercises/centralized-workflow-exercise#23]
The file laura_olivada_recipe.txt includes the instructions for a black olivada.

I would like to: 

-Change the name of  laura_olivada_recipe.txt to  laura_black_olivada_recipe.txt
-Add a new file with a recipe for green olivada named  laura_green_olivada_recipe.txt

**Comment by PotatisMaster** [comment]
Sounds great and tasty!
Comments:  laura_black_olivada_recipe.txt is a long filename, maybe have a shorter one?

**Comment by tpfau** [comment]
Would it make sense to give the whole recipe section some structure? e.g. a `recipe/` folder with subfolders like starter, main and dessert?

**Comment by l-gar-alv** [comment]
> 

Yes, thanks! Maybe I can have one single file laura_olivada_recipe.txt with both recipes.
